hack/00-osd-managed-cluster-config-integration.yaml.tmpl

```diff
@@ -2761,6 +2761,133 @@ objects:
           namespacesAllowedRegex: (^redhat-rhoami?$|^redhat-rhoami?-.*)
         subjectKind: Group
         subjectName: system:serviceaccounts:openshift-backplane-cssre
+- apiVersion: hive.openshift.io/v1
+  kind: SelectorSyncSet
+  metadata:
+    labels:
+      managed.openshift.io/gitHash: ${IMAGE_TAG}
+      managed.openshift.io/gitRepoName: ${REPO_NAME}
+      managed.openshift.io/osd: 'true'
+    name: backplane-cssre-rhoc-addon-connectors-operator
+  spec:
+    clusterDeploymentSelector:
+      matchLabels:
+        api.openshift.com/managed: 'true'
+        api.openshift.com/addon-connectors-operator: 'true'
+    resourceApplyMode: Sync
+    resources:
+    - apiVersion: rbac.authorization.k8s.io/v1
+      kind: ClusterRole
+      metadata:
+        name: backplane-cssre-rhoc-admins-project
```

Comment:
@pat-cremin it looks like the `backplane-cssre-rhoc-admins-project`is correct. @mrbarge

Reply:
I had done so, it must have been an older version that had make run against it( i am unable to run make without error). 

Re-pushed and waiting on @Nanyte25 to run make for me.

Reply:
@pat-cremin I have ran the make and push the hack file to you branch